src/Entity/Value.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
```

Comment:
What is purpose and benefit of this object encapsulation of all - even primitive - values?

This class is copied from Java :trollface:?

I don't see anything wrong with mixed being used where it makes sense and there is no type safety benefit of encapuslating mixed into this class (which also supports mixed and adds no extra logic besides storing the mixed). So am I missing something?

Reply:
`Value` "_type_" is used only inside of the library to explicitelly work with `Value` not just any `mixed` value. It is easier and more explicit to work with `Value` in services.